src/NServiceBus.Metrics/Extensions.cs

```diff
@@ -19,14 +20,18 @@ public static bool TryGetTimeSent(this ReceivePipelineCompleted completed, out D
 
     public static bool TryGetMessageType(this ReceivePipelineCompleted completed, out string ProcessedMessageType)
     {
-        var headers = completed.ProcessedMessage.Headers;
+        return completed.ProcessedMessage.Headers.TryGetMessageType(out ProcessedMessageType);
```

Comment:
It's pedantic but should processedMessageType be camelCased (as opposed to PascalCased)?